amazon/ion/simpleion.py

```diff
@@ -297,7 +303,7 @@ def dumps(obj, imports=None, binary=True, sequence_as_stream=False, skipkeys=Fal
     return ret_val
 
 
-def load(fp, catalog=None, single_value=True, encoding='utf-8', cls=None, object_hook=None, parse_float=None,
+def load_original(fp, catalog=None, single_value=True, encoding='utf-8', cls=None, object_hook=None, parse_float=None,
```

Comment:
I renamed load to load_original so:

1. prevent confusion from the same name API. (pure python load api and the final load api which users use)
2. We may use both pure python load and c extension load in a test.

So does dump.

Reply:
Instead of load/dump_original let's use load/dump_pure_python